src/connector/Cargo.toml

```diff
@@ -106,6 +106,7 @@ redis = { version = "0.24.0", features = ["aio","tokio-comp","async-std-comp"] }
 regex = "1.4"
 reqwest = { version = "0.11", features = ["json"] }
 risingwave_common = { workspace = true }
+risingwave_hummock_sdk = { workspace = true }
```

Comment:
Maybe this is the reason why UT will timeout? IIUC, `test_future_delivery_manager_compress_chunk` doesn't need to rely on `EpochWithGap`. Should we revert this and the changes in `test_future_delivery_manager_compress_chunk`? cc @wenym1 to confrm.